capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisFragmentImpl.java

```diff
@@ -90,13 +92,13 @@ public void setListener(@NonNull final AnalysisFragmentListener listener) {
 
     @Override
     void showScanAnimation() {
-        mProgressActivity.setVisibility(View.VISIBLE);
         mAnalysisMessageTextView.setVisibility(View.VISIBLE);
+        injectedLoadingIndicatorContainer.getInjectedViewAdapter().onVisible();
```

Comment:
[Request] The injected view adapter might be null. I would recommend to do a null check before calling `onVisible()`.